### PR TITLE
fixed AuthenticationService and UserService methods

### PR DESCRIPTION
**Pull request by kvjohnny**

### CODE REVIEW

File: src/main/java/mate/academy/service/AuthenticationService.java [hunk 1]
```diff
@@ -1,16 +1,18 @@
...
+     *
+     * @param email    - user's email
      * @param password - user's password
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
-        return false;
+        return findByEmail(email) != null
+                && findByEmail(email).getPassword().equals(password);
```
Comment (Sanderfuck) [inline src/main/java/mate/academy/service/AuthenticationService.java:16]: You duplicate 2 times - findByEmail(email), save it to variable

File: src/main/java/mate/academy/service/AuthenticationService.java [hunk 2]
```diff
@@ -1,16 +1,18 @@
 package mate.academy.service;
 
-public class AuthenticationService {
+public class AuthenticationService extends UserService {
```
Comment (Sanderfuck) [inline src/main/java/mate/academy/service/AuthenticationService.java:3]: You can`t extends from UserService

**Review by Sanderfuck — CHANGES_REQUESTED** [review]

File: src/main/java/mate/academy/service/AuthenticationService.java [hunk 3]
```diff
@@ -1,16 +1,22 @@
...
-     * @param email - user's email
+     *
+     * @param email    - user's email
      * @param password - user's password
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
-        return false;
+        User byEmail = userService.findByEmail(email);
```
Comment (okuzan) [inline src/main/java/mate/academy/service/AuthenticationService.java:19]:
```suggestion
        User user = userService.findByEmail(email);
```
bad naming, doesn't tell us what it stores

**Review by okuzan — CHANGES_REQUESTED** [review]

File: src/main/java/mate/academy/model/User.java [hunk 4]
```diff
@@ -1,8 +1,8 @@
 package mate.academy.model;
 
 public class User {
-    private String email;
-    private String password;
+    private final String email;
+    private final String password;
```
Comment (Sanderfuck) [inline src/main/java/mate/academy/model/User.java:5]: You don`t need it

File: src/main/java/mate/academy/service/AuthenticationService.java [hunk 5]
```diff
@@ -1,16 +1,22 @@
...
-     * @param email - user's email
+     *
+     * @param email    - user's email
      * @param password - user's password
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
-        return false;
+        User findByEmail = userService.findByEmail(email);
```
Comment (Sanderfuck) [inline src/main/java/mate/academy/service/AuthenticationService.java:19]: Change name "user" instead "findByEmail", your naming not informative

**Review by Sanderfuck — CHANGES_REQUESTED** [review]

**Review by olekskov — APPROVED** [review]
Good job

**Comment by kvjohnny** [comment]
Thank you so much!

**Pull request closed without merging**